kubos/use.py

```diff
@@ -19,13 +19,18 @@
 from kubos.utils.constants import  KUBOS_SRC_DIR
 
 def addOptions(parser):
-    parser.add_argument('set_version', nargs=1, help='Set a specific version of the KubOS modules to build your projects against.')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--branch', nargs='?', help='Set the branch flag to specify to checkout a branch, not a tag')
```

Comment:
1. Add a short version of this (-b?)
2. Make the positional argument required.  If --branch is specified, the user should be required to specify a branch name.